### PR TITLE
Add more explicit wait in test

### PR DESCRIPTION
**Pull request by MCGallaspy**
Does what it says. Also add something to .gitignore that I've been meaning to do for a while.

### CODE REVIEW

File: kalite/distributed/static/js/distributed/contentrec/hbtemplates/content-nextsteps-lesson.handlebars [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 <div class="row" class="content-nextsteps-lesson">
-	<a href="/learn/{{ path }}">
+	<a href="/learn/{{ path }}" id="content-nextsteps-lesson-link">
```
Comment (rtibbles) [inline kalite/distributed/static/js/distributed/contentrec/hbtemplates/content-nextsteps-lesson.handlebars]: This is a repeated element in the page, we shouldn't use an ```id``` attribute here - can you change it to a class?

**Comment by MCGallaspy** [comment]
@rtibbles added another commit. See commit message for explanation.

**Comment by rtibbles** [comment]
Looks good, let's see what the Circle master says!

**Comment by rtibbles** [comment]
No test failures I would not expect. Merging.

**Pull request merged**